apps/prairielearn/src/pages/instructorAssessmentOverrides/instructorAssessmentOverrides.js

```diff
@@ -0,0 +1,78 @@
+const error = require('@prairielearn/error');
+const express = require('express');
+const asyncHandler = require('express-async-handler');
+const router = express.Router();
+const { loadSqlEquiv } = require('@prairielearn/postgres');
+const sqldb = require('@prairielearn/postgres');
+const sql = loadSqlEquiv(__filename);
+
+router.get(
+  '/',
+  asyncHandler(async (req, res) => {
+    const params = {
+      assessment_id: res.locals.assessment.id,
+      current_student_uid: res.locals.user.uid,
+      current_assessment_title: res.locals.assessment.title,
+    };
+    const result = await sqldb.queryAsync(sql.select_assessment_access_policy, params);
+    res.locals.policies = result.rows;
+    res.render(__filename.replace(/\.js$/, '.ejs'), res.locals);
+  }),
+);
+
+router.post(
+  '/',
+  asyncHandler(async (req, res) => {
+    if (req.body.__action === 'add_new_override') {
+      const params = {
+        assessment_id: res.locals.assessment.id,
+        created_by: res.locals.user.uid,
+        credit: req.body.credit,
+        end_date: new Date(req.body.end_date),
+        group_name: req.body.group_name || null,
+        note: req.body.note || null,
+        start_date: new Date(req.body.start_date),
+        student_uid: req.body.student_uid,
```

Comment:
You'll want to enforce that only `group_name` or `student_uid` was provided, and not both. You should also enforce that `group_name` is accepted only for group assessments, and that `student_uid` is provided only for non-group assessments.